compose-files/creds_setup.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
Would you move this to 'Docker/compose-files` rather than setup a root level directory?